easy-ui-react/src/Tooltip/Tooltip.module.scss

```diff
@@ -0,0 +1,74 @@
+@use "../styles/common" as *;
+
+.Tooltip {
+  @include component-token("tooltip", "arrow-size", "1rem");
+  @include component-token("tooltip", "max-width", 320px);
+  @include component-token("tooltip", "shadow", theme-token("shadow.overlay"));
+  @include component-token(
+    "tooltip",
+    "background",
+    theme-token("color.surface.overlay")
+  );
+  @include component-token(
+    "tooltip",
+    "border_radius",
+    design-token("shape.border_radius.base")
```

Comment:
**question**: out of curiosity, is there a benefit to wrapping design tokens in component tokens?